src/powershellScripts.ts

```diff
@@ -79,12 +87,43 @@ function fold ($children, $Block) {
     $hashset.Clear() | Out-Null
 }
 
-$found = Discover-Test -Path $Path
+function Get-PesterDiscoveries {
```

Comment:
Nit: Plural nouns should not be used in function naming. I'd suggest `Get-FileSuite`. Actually `Get-FileSuiteOld`, because you plan to deprecate it later.